api/src/main/java/marquez/db/JobDao.java

```diff
@@ -231,6 +231,10 @@ default PGobject toJson(Set<DatasetId> dataset, ObjectMapper mapper) {
     }
   }
 
+  /*
+   Note: following SQL never executes. There is database trigger on `jobs_view` that replaces following SQL
```

Comment:
I might leave off that first sentence - it's a bit inaccurate. As far as the Jdbi client is concerned, this is the SQL that is executed. I might phrase it as 

> Inserts to the `jobs_view` are supported by the `rewrite_jobs_fqn_table` plpgsql function. Code of that function is at R__1 migration file.

Reply:
Ah sorry @collado-mike! I enabled auto merge